emu/src/cpu/instruction.rs

```diff
@@ -689,7 +693,25 @@ impl ThumbModeInstruction {
 
                 format!("{instr} {{{registers}}}")
             }
-            Self::MultipleLoadStore => "".to_string(),
+            Self::MultipleLoadStore {
+                load_store,
+                base_register,
+                register_list,
+            } => {
+                let instr = match load_store {
+                    LoadStoreKind::Load => "LDMIA",
+                    LoadStoreKind::Store => "STMIA",
+                };
+
+                let mut registers = String::new();
+                for i in 0..=7 {
+                    if register_list.get_bit(i) {
+                        registers.push_str(&format!("R{}, ", i));
```

Comment:
Like this we would have `{R0, R1, }`, right? (I mean at the end we would have an additional comma).

Also, to be consistent with the datasheet we should use `-` to separate registers :P

Reply:
I know... Right now I took a shortcut :D we can leave as good first issue :) (another instruction has this problem) :D